CHANGELOG

```diff
@@ -1,3 +1,9 @@
+4.1.5 (pre)
```

Comment:
I follow the convention of using `4.1.5.dev0` here, which matches the corresponding PEP IIRC and is understood by `pip`. The version number should also be changed accordingly in `setup.py`.

Although I tried to follow the workflow changing the version to the next `dev` version directly after the release, so the individual PRs do not need to take care of this. But I forgot to do that here.